wordpress-mobile/WordPress-Android
# Fix RTL in refreshed Stats

Fixes #8581 
Fixes RTL issues in the new Stats after the refresh
* Fix of the Empty block - No data yet was aligned to the left
* Fix of the Labels - they were stuck together in the middle of the block
* Fix of the Link - the link was aligned to the left

To test:
* Set "Force RTL" in dev's settings
* Open Stats
* Check that everything is aligned to the right correctly
